openqa-label-known-issues

```diff
@@ -136,31 +138,33 @@ investigate_issue() {
     echo "$reason" >> "$out"
     if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]]; then
         # if we can not even access the page it is something more critical
-        handle_unreachable "$testurl" "$out"
-        # Assume autoinst-log.txt still not found but process job anyway due to reason var
-        if [[ -n $reason ]] && [[ $curl_output = "404" ]] && [[ $reason != null ]]; then
-            label_on_issues_from_issue_tracker "$id"
+        handle_unreachable "$testurl" "$out" || return
+        # If we something other than a 404, we should not continue
+        [[ $curl_output != 404 ]] && return
         # Checking timestamp
-        elif [[ $(date -uIs -d '-14days') > $(grep timeago "$out" | hxselect -s '\n' -c '.timeago::attr(title)') ]]; then
+        if [[ $(date -uIs -d '-14days') > $(grep timeago "$out" | hxselect -s '\n' -c '.timeago::attr(title)') ]]; then
```

Comment:
To me it looks like `out` contains the contents of autoinst-log.txt and the reason. This code is looking for a certain HTML element in `out` which looks wrong. Have you actually tested this? I don't see that the unit tests below cover anything containing `timeago`.

Considering you're not really introducing this line of code here it is probably not that important - but I cannot really make sense of the code.

Reply:
The content in `$out` is overwritten in `handle_unreachable`:
```
if ! curl "${curl_args[@]}" -s "$testurl" -o "$out"; then
```

Reply:
Ah, *of course* :-)

Reply:
This might actually be another thing that is now kind of broken after the #335.
If the logfile is overwritten, then trying to label it doesn't make sense anymore.
Maybe `handle_unreachable` and this check here shuold use a different filename.
Originally this check was inside of `handle_unreachable`, and the code didn't continue after that.

Reply:
@b10n1k Is this case covered in unit tests?

Reply:
@b10n1k Do you see unresolved comments? 🙃

Reply:
As we discusssed, this might be a thing but it is more that just change the name of the file. So I left it out of this PR and it is something to look at later